tests/test_env.py

```diff
@@ -27,7 +28,9 @@ def test_isolation():
 
 
 @pytest.mark.isolated
-def test_isolated_environment_install(mocker):
+def test_isolated_environment_install(mocker, caplog):
+    # ensure the log code is used
+    caplog.set_level(logging.INFO)
```

Comment:
Instead of doing this, could you add a test in `test_projectbuilder.py` to verify the default `ProjectBuilder.log` implementation is correct?

Reply:
Sorry, I end up doing it myself as I has the time and I want this fixed ASAP :sweat_smile: